eventbot.py

```diff
@@ -0,0 +1,155 @@
+from collections import namedtuple
+from config import getLogger
+from bs4 import BeautifulSoup
+import requests
+import datetime
+import string
+import json
+from pytz import timezone, utc
+from dateutil.parser import parse
+from bots import RedditBot
+from config.bot_config import get_subreddits
+
+# region constants
+BASE_URL = "http://www.upressonline.com/fauevents/"
+TABLE_ROW = "{title} | {date} | {description}\n"
+HYPERLINK = "[{text}]({url})"
+HEADER_DIVIDER = "---|---|----\n"
+TABLE_HEADER = TABLE_ROW.format(title='Title', date='Date', description='Description') + HEADER_DIVIDER
+# endregion
+
+logger = getLogger()
+
+
+class EventBot(RedditBot):
+    def __init__(self, user_name, *args, **kwargs):
+        super(EventBot, self).__init__(user_name=user_name, *args, **kwargs)
+        self.base_url = BASE_URL
+        self.subreddits = get_subreddits()
+        self.post_title = "Event Calendar"
+
+    @staticmethod
+    def has_event_passed(event_json):
+        """
+        Takes the relevant values out of the event JSON and creates a simpler dictionary
```

Comment:
This is the wrong description. It's actually a copy of the docstring from `_get_event_dict`. Just make up a new description for this function.